io/disk/ssd/nvmetest.py

```diff
@@ -126,6 +127,127 @@ def reset_controller_sysfs(self):
             % self.device.split("/")[-1]
         return process.system(cmd, shell=True, ignore_status=True)
 
+    def get_max_ns_count(self):
+        """
+        Returns the maximum number of namespaces supported
+        """
+        output = self.get_id_ctrl_prop('nn')
+        if output:
+            return int(output)
+        return 1
+
+    def get_total_capacity(self):
+        """
+        Returns the total capacity of the nvme controller. 0 if not found.
```

Comment:
can Doc string will be like 
`Returns the total capacity of the nvme controller otherwise return   0 (if not found).`

Reply:
Sure, will change it.